index.js

```diff
@@ -5,9 +5,10 @@ function priceFormat(amountInCents, currencySymbol) {
   return currencySymbol + (amountInCents / 100).toFixed(2).replace(/\B(?=(\d{3})+(?!\d))/g, ',');
 }
 
-function priceFormatMultiCurrency(unitAmount, currencyCode) {
+function priceFormatMultiCurrency(unitAmount, currencyCode, locale) {
   currencyCode = currencyCode || 'usd';
-  const intlFormat = new Intl.NumberFormat(undefined, {
+  locale = locale.replace('_', '-');
```

Comment:
@nate-gehringer Wondering your opinion on this if it should be handled before values are even passed into the package functions?

Reply:
Yeah—for now, I think leaving it in place here is ok, but I agree that it’d be better to standardize on the `Intl` convention with hyphens further upstream – such as before persisting locale codes into the database.

Also—could you please update `'usd'` above to `'USD'`?